### ISSUE TITLE
Long lines are printed as a "K"

### ISSUE THREAD

**Issue by wch** [rstudio/rstudio#6995]
### System details

    RStudio Edition : desktop
    RStudio Version : 1.4.368
    OS Version      : macOS Catalina 10.15.4
    R Version       : 3.6.1

### Steps to reproduce the problem

Run the following:

```R
cat(paste(rep(".", 10240), collapse = ""))
cat(paste(rep(".", 10241), collapse = ""))
```

This is what it looks like:
![image](https://user-images.githubusercontent.com/86978/83195292-a430fd00-a0ff-11ea-8207-d5e940afaf28.png)

However, when I copy and paste into another text editor, the "K" turns into the (correct) long string of `.....` characters.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by aronatkins** [comment]
Also see this issue with RStudio 1.3.958.

**Comment by kevinushey** [comment]
Unfortunately cannot reproduce (macOS 10.15.4):

<img width="456" alt="Screen Shot 2020-05-29 at 12 10 26 PM" src="https://user-images.githubusercontent.com/1976582/83296288-62f32880-a1a5-11ea-9ee9-f3efe8635a0f.png">

**Comment by aronatkins** [comment]
Confirmed that the "K" appears here with 10241 characters. Fewer characters are printed.

macOS 10.15.4, RStudio 1.3.958, and R version 3.6.3 (2020-02-29)

**Comment by kevinushey** [comment]
I can reproduce now -- had to switch my font to Menlo.

**Comment by aronatkins** [comment]
Please keep this alive.

**Comment by gtritchie** [comment]
I can still repro this in latest release, and latest daily. It does not repro in a local Electron build. Will move this to ready for test.

**Comment by gtritchie** [comment]
Confirmed this is fixed in Electron, tried 2022.06.0-daily+244 on macOS 12.3.
